dapp/src/components/issuance/CurrencyType.jsx

```diff
@@ -1,32 +1,60 @@
-  { 'text': 'One time issued token', 'img': OneTimeIssuer, value: 'basic', tooltipText: `Those types of tokens could be minted once and their supply is fixed. This allows to create assets which are digitally scarce and can be produced in limited supply. Usually to achieve a higher level of protection and guarantee no one can create more coins of this type can protect its value and decentralization.` },
-  { 'text': 'Use existing token', 'img': DefaultIcon, value: 'existingToken', tooltipText: `Any existing coin that is ERC-20 compatible could be integrated into your community. We currently support DAI and will be adding more stable-coins very soon. Any stable coin that is operating on the Ethereum network could be plugged into your community. ` }
+// const communityTypes = [
+//   { 'text': 'Mintable/Burnable token', 'img': MintableBurnable, value: 'mintableBurnable', tooltipText: `This allows community manager to mint or burn tokens and fully control the token supply. Those types of tokens are used for credit-like or reward systems that can be used by any organization.` },
+//   { 'text': 'One time issued token', 'img': OneTimeIssuer, value: 'basic', tooltipText: `Those types of tokens could be minted once and their supply is fixed. This allows to create assets which are digitally scarce and can be produced in limited supply. Usually to achieve a higher level of protection and guarantee no one can create more coins of this type can protect its value and decentralization.` },
+//   { 'text': 'Use existing token', 'img': DefaultIcon, value: 'existingToken', tooltipText: `Any existing coin that is ERC-20 compatible could be integrated into your community. We currently support DAI and will be adding more stable-coins very soon. Any stable coin that is operating on the Ethereum network could be plugged into your community. ` }
+// ]
+
+const CommunityTypes = [
+  { label: 'Mintable/Burnable token', icon: MintableBurnable, value: 'mintableBurnable', tooltipText: `This allows community manager to mint or burn tokens and fully control the token supply. Those types of tokens are used for credit-like or reward systems that can be used by any organization.` },
+  { label: 'One time issued token', icon: OneTimeIssuer, value: 'basic', tooltipText: `Those types of tokens could be minted once and their supply is fixed. This allows to create assets which are digitally scarce and can be produced in limited supply. Usually to achieve a higher level of protection and guarantee no one can create more coins of this type can protect its value and decentralization.` }
+  // { label: 'Use existing token', icon: DefaultIcon, value: 'existingToken', tooltipText: `Any existing coin that is ERC-20 compatible could be integrated into your community. We currently support DAI and will be adding more stable-coins very soon. Any stable coin that is operating on the Ethereum network could be plugged into your community. ` }
```

Comment:
can we delete this?

Reply:
I'll remove all src/components/issuance/ issuace folder.